In utils/frost-icon-svg, the attribute name of the option object being passed to `resolve.sync() ` is wrong.

# BUG REPORT

## Summary
In utils/frost-icon-svg, the attribute name of the `option` object being passed to `resolve.sync() ` is wrong. It is specified as `baseDir` when it should be `basedir`.

As a result, resolve() will use the caller's file dir as the basedir which is not reliable.

## Expected Behavior
There should never be an error on build.

## Actual Behavior
Seeing error on build `Cannot find module 'frost-app-bar' from '/Users/jscharf/Projects/emberjs/frost-app-bar/node_modules/ember-frost-core/utils'`

## Steps to Reproduce
Install any package that has ember-frost-core as dependency to app root and setup svg usage. (DON'T add ember-frost-core to app's package.json)

## Context

## Environment

--- a/utils/frost-icon-svg.js
+++ b/utils/frost-icon-svg.js
@@ -82,7 +82,7 @@ const getSourceDirectory = function(svgSourceDir) {
         || this.project.pkg.name !== this.moduleName() // ...in a nested addon
     )
     {
-        let packagePath = resolve.sync(this.moduleName(), {baseDir: this.project.root})
+        let packagePath = resolve.sync(this.moduleName(), {basedir: this.project.root})
         sourcePath = path.join(packagePath.substring(0, packagePath.lastIndexOf('/')), svgSourceDir)
     }
 

